lib/src/foundation/bloc.dart

```diff
@@ -81,7 +81,7 @@ class Bloc {
     _eventsController.close();
   }
 
-  void discoverFeatures({Iterable<String> steps}) {
+  void discoverFeatures({@required Iterable<String> steps}) {
```

Comment:
This should not be a named parameter. It is technically a public part of the API, but it is not exported in `feature_discovery.dart`. Hence, I am not calling this a breaking change and it should be fine.